app/presenters/content_item_presenter.rb

```diff
@@ -175,6 +175,13 @@ def facets
         display_as_result_metadata: true,
         filterable: false
       },
+      {
+        key: "people",
+        name: "People",
+        type: "text",
+        display_as_result_metadata: false,
```

Comment:
The metadata already contains quite a lot of information. Organisations, date and format are present on all items; people aren't so I think the right thing is to leave them off.